translate contributing.md to PT-BR (#184)
b2wads/grimorio-ui: CONTRIBUTING.md

# Contribuindo

O primeiro passo para contribuir para esse repositório é primeiro abrir uma Issue, ou qualquer outro método que envolva os proprietários desse projeto, antes de fazer qualquer alteração

## Processo para o Pull Request

> 1. Fork esse repositório

Reviewer: Aqui acho melhor usar a expressão:
```suggestion
1. Faça um fork desse repositório.
```

Além disso, pode colocar ponto final em todos os itens? :D Apenas para manter um padrão!

2. Confirme a instalação de todas as dependências
3. Atualize o arquivo Readme.md com os detalhes das alterações para a Interface, se for aplicável
4. Abra uma Pull Request com suas alterações na branch `develop`.

## Código de Conduta

### Nossa PROMESSA

No interesse de promover um ambiente aberto e acolhedor, nós, como

> contribuintes e Mainteners nos comprometemos com a participação em nosso projeto e

Reviewer: Aqui (por mais feio que seja :P) acho melhor botar mantenedores ao invés de mantainers!
```suggestion
contribuintes e mantenedores nos comprometemos com a participação em nosso projeto e
```

Author: Eu acho horrivel essa palavra traduzida! Mas vou mandar bala!

> No interesse de promover um ambiente aberto e acolhedor, nós, como
> contribuintes e Mainteners nos comprometemos com a participação em nosso projeto e
> nossa comunidade, que ela seja uma experiência livre de assédio para todos, independentemente da idade, corpo

Reviewer 2: Acho que tem uma vírgula depois de mantenedores. E eu sugeriria alterar um pouquinho uma parte. O que vocês acham?

```suggestion
Com o objetivo de promover um ambiente aberto e acolhedor, nós, como contribuintes e mantenedores, nos comprometemos a tornar a participação neste projeto e comunidade uma experiência livre de assédio para todos, independentemente da idade, corpo
```

Reviewer: Perfeito @dehmirandac2!

tamanho, deficiência, etnia, identidade e expressão de gênero, nível de experiência,
nacionalidade, aparência pessoal, raça, religião ou identidade sexual e
orientação.

### Nossos PADRÕES

Exemplos de comportamentos que contribuem para criar um ambiente positivo para a nossa comunidade incluem:

* Demonstrar empatia e bondade com as outras pessoas
* Respeitar opiniões, pontos de vista e experiências contrárias
* Dar e receber feedbacks construtivos de maneira respeitosa

> * Assumir responsabilidade, pedir desculpas às pessoas afetadas por nossos erros e aprender com a experiência

Reviewer: Vi que você adicionou esse item! Achei ótimo! Mas acabou omitindo um deles, sobre a linguagem inclusiva. Portanto sugiro:
```suggestion
* Usar uma linguagem inclusiva e receptiva
* Assumir responsabilidade, pedir desculpas às pessoas afetadas por nossos erros e aprender com a experiência
```

* Focar no que é melhor não só para nós individualmente, mas para a comunidade em geral

Exemplos de comportamentos inaceitáveis incluem:

* Uso de linguagem ou imagens sexualizadas, bem como o assédio sexual ou de qualquer natureza
* Comentários insultuosos/depreciativos e ataques pessoais ou políticos (*Trolling*)
* Assédio público ou privado
* Publicar informações particulares de outras pessoas, como um endereço de e-mail ou endereço físico, sem a permissão explícita delas
* Outras condutas que são normalmente consideradas inapropriadas em um ambiente profissional

### Nossas RESPONSABILIDADES

> A liderança do Projeto são responsáveis ​​por esclarecer os padrões de comportamento aceitável e devem tomar as ações corretivas adequadas e justas em resposta a quaisquer instâncias de comportamento inaceitável.

Reviewer: Com relação a alguns plurais, acho que ficaria melhor assim:

```suggestion
A liderança do Projeto é responsável ​​por esclarecer os padrões de comportamento aceitáveis e devem tomar as ações corretivas adequadas e justas em resposta a quaisquer instâncias de comportamento inaceitável.
```
